src/java.desktop/windows/native/libawt/windows/awt_Taskbar.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2016, 2025, Oracle and/or its affiliates. All rights reserved.
```
A comma after 2025 is required.